tests/apollo/test_skvbc_st_reply_size.py

```diff
@@ -0,0 +1,107 @@
+# notices and license terms. Your use of these subcomponents is subject to the
+# terms and conditions of the subcomponent's license, as noted in the LICENSE
+# file.
+import os.path
+import unittest
+from os import environ
+
+import trio
+
+from util import skvbc as kvbc
+from util.bft import with_trio, with_bft_network, KEY_FILE_PREFIX
+
+
+def build_cmd(reply_size=None, page_size=None, num_pages=None):
+    def start_replica_cmd(builddir, replica_id):
+        """
+        Return a command that starts an skvbc replica when passed to
+        subprocess.Popen.
+        """
+        statusTimerMilli = "500"
+        path = os.path.join(builddir, "tests", "simpleKVBC", "TesterReplica", "skvbc_replica")
+        cmd = [
+          path,
+          "-k", KEY_FILE_PREFIX,
+          "-i", str(replica_id),
+          "-s", statusTimerMilli,
+        ]
+        if os.environ.get('BUILD_ROCKSDB_STORAGE', "").lower() in set(["true", "on"]):
+          cmd.append("-p")
+
+        if reply_size is not None:
+          cmd.extend(["--max-reply-message-size", str(reply_size)])
+        if page_size is not None:
+          cmd.extend(["--max-num-of-reserved-pages", str(num_pages)])
+        if page_size is not None:
+          cmd.extend(["--size-of-reserved-page", str(page_size)])
+
+        return cmd
+    return start_replica_cmd
+
+class SkvbcStReplySize(unittest.TestCase):
+
+    __test__ = False  # so that PyTest ignores this test scenario
+
+    async def state_transfer_after_first_checkpoint(self, bft_network):
+        """
+        Test that state transfer completes for a delayed node.
+
+        Start 3/4 nodes, wait for the first checkpoint to happen. Start the
+        remaining node and make sure that state transfer completes.
+        """
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+
+        late_node = 3
+        initial_nodes = bft_network.all_replicas(without={late_node})
+        bft_network.start_replicas(initial_nodes)
+
+        await skvbc.fill_and_wait_for_checkpoint(
+            initial_nodes,
+            checkpoint_num=1,
+            verify_checkpoint_persistency=False
+        )
+        checkpoint = await bft_network.wait_for_checkpoint(replica_id=0)
+        self.assertEqual(checkpoint, 1)
+
+        bft_network.start_replica(late_node)
+        await bft_network.wait_for_state_transfer_to_start()
+        await bft_network.wait_for_state_transfer_to_stop(0, late_node)
+
+    # Note, we only run 4 skvbc test replicas. Each replica is configured to
+    # accomodate up to 100 client proxies and hence the formula for maximum
+    # number of reserved pages becomes: 100 * 4 * reply_size / page_size
+
+    @with_trio
+    @with_bft_network(start_replica_cmd=build_cmd(reply_size=1<<13,
+                                                  page_size=1<<12,
+                                                  num_pages=800),
+                      selected_configs=lambda n,f,c: n==4 and f==1 and c==0)
```

Comment:
You could simply say: `selected_configs=lambda n,f,c: n==4`, or even `selected_configs=lambda n,_,_: n==4`